### ISSUE TITLE
Developer end-user's public key update bug

### ISSUE THREAD

**Issue by dang03** [sonata-nfv/son-gkeeper#870]
Gatekeeper API returns error 500 when updating end-user (developer) public key:
[users_controller.rb#L208](https://github.com/sonata-nfv/son-gkeeper/blob/master/son-gtkapi/routes/users_controller.rb#L208)
```
D, [2017-09-13 14:06:36#9] DEBUG -- GtkApi:: PATCH /api/v2/users/:username/user-public-key: entered with username=demo and body={"public_key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAtogbvZ4lvLrXC/R5blwj\n3j23jXQNXJts+RqiFcyAu03n9sbgNKPeesW8NSieFCVXqPbt2g2Itabu0i3KzuQO\nMCTv27QaCamEE3QVNxRluFWaWQvA8fb8HvBpM0h6rij0logwIavlcD9s3pt0/FHC\nr4j4PpHRlUKvs4ZjsRtqmgHJPzkpvGQ9eEBh+uCylNqfYaReWYRF/opTbbVpJ7/a\neVRY01fSaSz64XawcyWucsbUb/ZogzwFL7APqM9EQ4AmSsposKguI5bU+5b/+NvT\nGE91U5OIGBbDtszqRxahVaiTH9N1SAjTxmSqC5Hu4txLBZ5S3shaL3N2bQHnfAGY\n6QIDAQAB"}
2017-09-13 14:06:36 - NameError - undefined local variable or method `began_at' for #<GtkApi:0x00564f59011978>:
	/app/routes/users_controller.rb:208:in `block (2 levels) in <class:GtkApi>'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1611:in `call'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1611:in `block in compile!'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:975:in `[]'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:975:in `block (3 levels) in route!'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:994:in `route_eval'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:975:in `block (2 levels) in route!'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1015:in `block in process_route'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1013:in `catch'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1013:in `process_route'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:973:in `block in route!'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:972:in `each'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:972:in `route!'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1085:in `block in dispatch!'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1067:in `block in invoke'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1067:in `catch'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1067:in `invoke'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1082:in `dispatch!'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:907:in `block in call!'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1067:in `block in invoke'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1067:in `catch'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1067:in `invoke'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:907:in `call!'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:895:in `call'
	/usr/local/bundle/gems/rack-1.6.8/lib/rack/session/abstract/id.rb:225:in `context'
	/usr/local/bundle/gems/rack-1.6.8/lib/rack/session/abstract/id.rb:220:in `call'
	/usr/local/bundle/gems/rack-protection-1.5.3/lib/rack/protection/xss_header.rb:18:in `call'
	/usr/local/bundle/gems/rack-protection-1.5.3/lib/rack/protection/path_traversal.rb:16:in `call'
	/usr/local/bundle/gems/rack-protection-1.5.3/lib/rack/protection/json_csrf.rb:18:in `call'
	/usr/local/bundle/gems/rack-protection-1.5.3/lib/rack/protection/base.rb:49:in `call'
	/usr/local/bundle/gems/rack-protection-1.5.3/lib/rack/protection/base.rb:49:in `call'
	/usr/local/bundle/gems/rack-protection-1.5.3/lib/rack/protection/frame_options.rb:31:in `call'
	/usr/local/bundle/gems/rack-1.6.8/lib/rack/logger.rb:15:in `call'
	/usr/local/bundle/gems/rack-1.6.8/lib/rack/commonlogger.rb:33:in `call'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:219:in `call'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:212:in `call'
	/usr/local/bundle/gems/rack-1.6.8/lib/rack/head.rb:13:in `call'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:182:in `call'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:2013:in `call'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1487:in `block in call'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1787:in `synchronize'
	/usr/local/bundle/gems/sinatra-1.4.7/lib/sinatra/base.rb:1487:in `call'
	/usr/local/bundle/gems/puma-3.6.2/lib/puma/configuration.rb:225:in `call'
	/usr/local/bundle/gems/puma-3.6.2/lib/puma/server.rb:578:in `handle_request'
	/usr/local/bundle/gems/puma-3.6.2/lib/puma/server.rb:415:in `process_client'
	/usr/local/bundle/gems/puma-3.6.2/lib/puma/server.rb:275:in `block in run'
	/usr/local/bundle/gems/puma-3.6.2/lib/puma/thread_pool.rb:116:in `call'
	/usr/local/bundle/gems/puma-3.6.2/lib/puma/thread_pool.rb:116:in `block in spawn_thread'
E, [2017-09-13 14:06:36#9] ERROR -- : {"error":{"code":500,"message":"#<NameError: undefined local variable or method `began_at' for #<GtkApi:0x00564f59011978>>"}}
10.30.0.110 - - [13/Sep/2017:14:06:36 +0000] "PATCH /api/v2/users/demo/user-public-key HTTP/1.1" 500 125 0.0035
```

**Comment by jbonnet** [comment]
Hey, @dang03, I think it has been fixed by PR #871. Can you please try again and let me know?
Thanks,

**Comment by dang03** [comment]
@jbonnet 
Yes, it is fixed, works fine now.

```
Updating User Public Key...
r.status_code= 200
```
